### ISSUE TITLE
Failed to backup: Download shard 232 failed copy backup to file (v1.3.2)

### ISSUE THREAD

**Issue by hongquan** [influxdata/influxdb#8717]
Since I upgraded InfluxDB to v1.3.2, I cannot do backup:

```sh
$ influxd backup -database plantinghouse /tmp/ts_pt
2017/08/18 09:25:43 backing up db=plantinghouse since 0001-01-01 00:00:00 +0000 UTC
2017/08/18 09:25:43 backing up metastore to /tmp/ts_pt/meta.00
2017/08/18 09:25:43 backing up db=plantinghouse rp=autogen shard=232 to /tmp/ts_pt/plantinghouse.autogen.00232.00 since 0001-01-01 00:00:00 +0000 UTC
2017/08/18 09:25:43 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (0)...
2017/08/18 09:25:44 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (1)...
2017/08/18 09:25:45 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (2)...
2017/08/18 09:25:46 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (3)...
2017/08/18 09:25:47 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (4)...
2017/08/18 09:25:48 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (5)...
2017/08/18 09:25:50 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (6)...
2017/08/18 09:25:51 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (7)...
2017/08/18 09:25:52 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (8)...
2017/08/18 09:25:53 Download shard 232 failed copy backup to file: err=<nil>, n=0.  Retrying (9)...
2017/08/18 09:25:54 backup failed: copy backup to file: err=<nil>, n=0
backup: copy backup to file: err=<nil>, n=0
```

When I reverted back to v1.3.1, it is Ok.

Machine:

```sh
$ uname -a
Linux beaglebone-black 4.1.18-ti-r56 #1 SMP Thu Mar 31 00:22:03 UTC 2016 armv7l GNU/Linux
```

**Comment by d01mf** [comment]
Same here:
Linux xyz 3.13.0-62-generic #102-Ubuntu SMP Tue Aug 11 14:29:36 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

**Comment by agarturbo** [comment]
Ditto:
Linux tigger10.qeauto.tllsc.net 3.10.0-514.21.1.el7.x86_64 #1 SMP Thu May 25 17:04:51 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

**Comment by jackroofan** [comment]
Seems the fix by [https://github.com/influxdata/influxdb/pull/8378/files](url) has been revert by this https://github.com/influxdata/influxdb/commit/18a02d50d71fe55f14c27e5911eba10aaf7e4e7d

**Comment by jmelus** [comment]
Same:
Linux oss-db-influx 3.10.0-514.10.2.el7.x86_64 #1 SMP Fri Mar 3 00:04:05 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

InfluxDB v1.3.3 (git: HEAD e37afaf09bdd91fab4713536c7bdbdc549ee7dc6)

**Comment by e-dard** [comment]
@hongquan @d01mf @jackroofan @agarturbo  @jmelus  this issue should be fixed in the `1.3.4` release. There should already be some release candidates available if you don't/can't wait for the release. I expect the release won't be too far away now, though.

Thanks all for the reports.

Closing the issue as it's a duplicate of https://github.com/influxdata/influxdb/issues/8677

**Comment by duyunw** [comment]
哎 BUG怎么经常让我遇到 从gitlab,zabbix,再到influxdb那个版本有问题，我就能遇到那个版本，也是醉了
